.github/workflows/dockerd.yml

```diff
@@ -0,0 +1,175 @@
+name: dockerd
+
+on:
+  schedule:
+    - cron: '0 10 * * *'
+  # TODO: add event to build on command in PR (e.g., /test-dockerd)
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Docker version'
+        required: true
+        default: '20.10.13'
+
+env:
+  REPO_SLUG_ORIGIN: "moby/buildkit:latest"
+  CACHE_GHA_SCOPE_IT: "dockerd-integration-tests"
+  CACHE_GHA_SCOPE_BINARIES: "dockerd-binaries"
+
+jobs:
+  base:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Expose GitHub Runtime
+        uses: crazy-max/ghaction-github-runtime@v1
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          driver-opts: image=${{ env.REPO_SLUG_ORIGIN }}
+          buildkitd-flags: --debug
+      -
+        name: Build ${{ env.CACHE_GHA_SCOPE_BINARIES }}
+        run: |
+          ./hack/build_ci_first_pass binaries
```

Comment:
Do we need these complexities if it is always a manual run anyway. I think it can be a single job without any "first_pass" or test matrix. It is already a single worker and rarely runs. Can use the cache from the main CI.